low-level/framework/sspl_reinit

```diff
@@ -51,7 +51,7 @@ usermod -a -G disk sspl-ll
 usermod -a -G disk sspl-ll
 
 # Update the sudoers file with the sspl-ll user and available commands
-SUDO_LINE="sspl-ll	ALL = NOPASSWD: /usr/sbin/smartctl, /usr/sbin/mdadm, /usr/bin/mount, /usr/bin/umount, /usr/sbin/swapon, /usr/sbin/swapoff, /usr/sbin/hdparm, /usr/bin/systemctl, /usr/sbin/wbcli, /usr/bin/ipmitool, /usr/bin/systemd-detect-virt, /bin/salt-call, /bin/tee, /usr/bin/facter, /usr/bin/provisioner"
```

Comment:
Why /bin/tee is removed?
PR#163 added this yesterday.  @thavanathan

Reply:
It was because of ny mistake. Added it now. Thanks for pointing this out